astropy/io/votable/tests/test_vo.py

```diff
@@ -819,40 +819,41 @@ def test_select_columns_binary(format_):
     assert table.colnames == ["string_test", "string_test_2", "unicode_test"]
 
 
-def table_from_scratch():
+def test_direct_fields_mutation():
```

Comment:
here the diff looks fishy because it's collapsing 2 independent changes
- I added a new test whose preamble is based on `table_from_scratch`
- I *removed* `table_from_scratch` when I found that it wasn't used anywhere

I could split this cleanup into its own PR if desired.